tensorflow/python/keras/utils/vis_utils.py

```diff
@@ -86,6 +103,12 @@ def model_to_dot(model,
     expand_nested: whether to expand nested models into clusters.
     dpi: Dots per inch.
     subgraph: whether to return a `pydot.Cluster` instance.
+    layer_range: range of layers passed as `list` or `Tensor` of `int` or `str`
+        of shape (2,) indicating the range of layers for which the `pyDot.Dot`
```

Comment:
Its quite hard to see what's the supported types here, could we split each of them into individual lines?

Reply:
done